src/Configurator.php

```diff
@@ -72,6 +72,10 @@ public function populateUpdate(RecipeUpdate $recipeUpdate): void
 
     public function unconfigure(Recipe $recipe, Lock $lock)
     {
+        if ('kbond/homepage-scaffold' === $recipe->getPackage()->getName()) {
```

Comment:
I could not figure out how to determine if the recipe is a pack or not here. `$recipe->getPackage()->getType === null`.